data_model/fees_executor/src/parameters.rs

```diff
@@ -0,0 +1,21 @@
+use alloc::{format, string::String, vec::Vec};
+
+use iroha_executor_data_model::parameter::Parameter as ExecutorParameter;
+use iroha_schema::IntoSchema;
+use iroha_data_model::prelude::*;
+use serde::{Deserialize, Serialize};
+
+#[derive(PartialEq, Eq, ExecutorParameter, Deserialize, Serialize, IntoSchema)]
+pub struct FeesOptions {
+    pub receiver: AccountId,
+    pub asset: AssetDefinitionId,
```

Comment:
```suggestion
    pub asset: AssetId,
```

I think you can also have just like this. Not sure if it's any better but think about it